contents/android/IntroToAndroid.md

```diff
@@ -0,0 +1,145 @@
+# Introduction to Android App Development
+
+Authors: [Lee Yan Hwa](https://github.com/leeyh20)
+
+- [Getting Started](#1-getting-started)
+    - [What is Android?](#11-what-is-android)
+    - [Develop your first app today!](#12-develop-your-first-app-today)
+
+- [Android is not magic](#2-android-is-not-magic)
+    - [App Components](#21-app-components)
+    - [Activity Lifecycle](#22-activity-lifecycle)
+    - [The Architecture of Android](#23-the-architecture-of-android)
+    - [Resources](#24-resources)
+    - [Testing](#25-testing)
+- [Why should I make native Android apps?](#3-why-should-i-make-native-android-apps)
+- [Further Readings](#4-further-readings)
+
+## 1. Getting Started
+
+### 1.1 What is Android?
+
+Android is a Linux-based mobile phone operating system (OS), created by Android Inc. which was then acquired by Google. It was launched in 2008. Not only does it power phones, it is already the OS of watches, televisions and even car stereo systems.
+
+The source code of Android is made available through the [Android Open Source Project](https://source.android.com/) (AOSP). The original AOSP code is used mainly on the Nexus and Pixel phones that are developed by Google itself. The AOSP code is customized and adapted by original equipment manufacturers (OEMs), such as Samsung and Sony, to run on their respective phones. There is a list of OEMs [here](https://www.android.com/certified/partners/). Android also has many versions, from Android Ice Cream Sandwich to the latest Android Oreo (as of March 2018). The Android ecosystem is quite [fragmented](https://www.androidauthority.com/android-version-distribution-748439/), with mainly 5 Android versions being installed in Android devices today.
+
```

Comment:
Negative things like fragmentation can be mentioned in a later section as challenges of Android development.